test/junit/scala/collection/SeqTest.scala

```diff
@@ -48,11 +48,11 @@ class SeqTest extends AllocationTest {
 
     assertEquals(0, Vector(0, 1).indexOfSlice(Vector.empty[Int]))
     assertEquals(0, Vector(0, 1).indexOfSlice(Vector.empty[Int], from = -2))
-    assertEquals(-1, Vector(0, 1).indexOfSlice(Vector.empty[Int], from = 2))
+    assertEquals(2, Vector(0, 1).indexOfSlice(Vector.empty[Int], from = 2))
```

Comment:
I would intuitively expect `indexOf...` functions to return only valid indices, or `-1`. The docs

> the first index `>= from` such that ... or `-1` ...

also sound that way to me.

Reply:
I'm a Pisces, so I could go either way.